.github/workflows/terraform-deploy.yml

```diff
@@ -44,12 +44,12 @@ jobs:
       - name: Terraform init
         uses: ./.github/actions/setup
       
-      - name: Terraform test
-        run: cd test && go test -v -timeout 3000s infra_test.go
-        continue-on-error: true
+      # - name: Terraform test
+      #  run: cd test && go test -v -timeout 3000s infra_test.go
+      #  continue-on-error: true
 
-      # - name: Terraform deploy
-      #   run: TF_VAR_grafana_admin_password=${{ env.PASSWD }} terraform apply --auto-approve
+      - name: Terraform deploy
+        run: TF_VAR_grafana_admin_password=${{ env.PASSWD }} terraform apply --auto-approve
```

Comment:
why `PASSWD` you're not shortening anything! its just confusing and not even `PASS` as any convention, please rename it to `PASSWORD` instead